vagrant-spk

```diff
@@ -640,22 +644,30 @@ def publish(args):
     finally:
         os.remove(temp_spk)
 
-def halt(args):
+def warn_deprecated(command_name):
+    sys.stderr.write("WARNING: `vagrant-spk {0}` is deprecated and may be removed in the future.\nPrefer `vagrant-spk vm {0}`.\n".format(command_name))
```

Comment:
+1 to ANSI red, if you'd add it.

Also I suggest you say, "will be removed on August 1, 2016" rather than "may be removed [some day who knows whatever you can ignore this warning]".

If you prefer, I can be the one to make changes according to the above suggestions.